main.js

```diff
@@ -446,6 +317,136 @@ onReady = () => {
     }
 }; /* onReady() */
 
+// initialise the db
```

Comment:
This is core logic, which when changes can break the whole app. So it needs to be clear what changed, or you really know what you are doing and build that core logic. ;)